keyboards/dweebsmini/readme.md

```diff
@@ -0,0 +1,15 @@
+# dweebsmini
+
+![dweebsmini](https://i.redd.it/7oa8lp2q6mr31.jpg)
+
+This is a split Ortho keyboard, inspired by the CRKBD and the ErgoDash Mini
+
+Keyboard Maintainer: [KeyboardDweebs](HTTPS://KeyboardDweebs.com)  
```

Comment:
put extra space or <br> behind the next 3 lines or the document won't render correctly. Thanks